readme.md

```diff
@@ -17,6 +17,13 @@ Install `yo`, `grunt-cli`, `bower`, `generator-angular` and `generator-karma`:
 npm install -g grunt-cli bower yo generator-karma generator-angular
 ```
 
+Install Ruby (required in order to install compass which is used by this generator)
```

Comment:
Ok... but new users following the existing instructions (which don't document how to deselect sass) won't get their application launched until Ruby+compass is installed.   Are you suggesting that the mention of Ruby+compass installation be merely re-worded, or that it does not need to be raised at all ?